ons_alpha/jinja2/component_overrides/footer/_macro.njk

```diff
@@ -0,0 +1,218 @@
+{% macro onsFooterNew(params) %}{# This is an override of ONS "components/footer/_macro.njk" #}
```

Comment:
@ababic this overrides the design system template in order to add classes more granularly than the options allow, and in order to sort things like icons with hard-coded fills.